File: promocode-issue/src/main/java/com/voshodnerd/promocode/issue/repository/CampaignRepository.java
package com.voshodnerd.promocode.issue.repository;

import com.voshodnerd.promocode.issue.model.Campaign;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface CampaignRepository extends JpaRepository<Campaign, Long> {
}

File: promocode-issue-kotlin/src/main/kotlin/com/voshodnerd/promocode/issue/repository/CampaignRepository.kt
package com.voshodnerd.promocode.issue.repository;

import com.voshodnerd.promocode.issue.model.Campaign;
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository;

@Repository
interface CampaignRepository : JpaRepository<Campaign, Long>
